src/async-components/views/dialogs/eventindex/DisableEventIndexDialog.js

```diff
@@ -55,23 +52,24 @@ export default class ManageEventIndex extends React.Component {
 
     render() {
         const BaseDialog = sdk.getComponent('views.dialogs.BaseDialog');
-        const InlineSpinner = sdk.getComponent('elements.InlineSpinner');
+        const Spinner = sdk.getComponent('elements.Spinner');
+        const DialogButtons = sdk.getComponent('views.elements.DialogButtons');
 
         return (
             <BaseDialog className='mx_ManageEventIndexDialog'
                 onFinished={this.props.onFinished}
                 title={_t("Are you sure?")}
             >
-            {_t("If disabled, messages form encrypted rooms won't appear in search results")}
-            <div className="mx_Dialog_buttons">
-                <AccessibleButton kind="secondary" onClick={this.props.onFinished}>
-                    {_t("Cancel")}
-                </AccessibleButton>
-                <AccessibleButton kind="danger" disabled={this.state.disabling} onClick={this._onDisable}>
-                    {_t("Disable")}
-                </AccessibleButton>
-                {this.state.enabling ? <InlineSpinner /> : <div />}
-            </div>
+                {_t("If disabled, messages from encrypted rooms won't appear in search results.")}
+                {this.state.disabling ? <Spinner /> : <div />}
+                <DialogButtons
+                    primaryButton={_t('Disable')}
+                    onPrimaryButtonClick={this._onDisable}
+                    primaryButtonClass="danger"
+                    onCancel={this.props.onFinished}
+                    disabled={this.state.disabling}
+                >
```

Comment:
If you're keeping the spinner outside the button row like this, and thus there's no children for `DialogButtons`, you can use a self-closing tag instead here.

Reply:
I tried to keep the spinner inline, but the buttons would then be one above the other. It seems that `DialogButtons` doesn't expect any content. Will remove the separate closing tag.